openfisca_france/model/caracteristiques_socio_demographiques/logement.py

```diff
@@ -57,7 +50,10 @@ class proprietaire_proche_famille(Variable):
     entity_class = Familles
     label = u"Le propriétaire du logement a un lien de parenté avec la personne de référence ou son conjoint"
 
-
+class habite_chez_parents(Variable):
```

Comment:
In this case, there is usually only one person in the family, and we have no information about the parents. We just need to know if the person is geographically independent.